lightly/cli/_helpers.py

```diff
@@ -43,8 +43,8 @@ def fix_hydra_arguments(config_path: str = "config", config_name: str = "config"
     hydra_args = {"config_path": config_path, "config_name": config_name}
 
     try:
-        if version_compare(hydra.__version__, "1.1.2") > 0:
-            hydra_args["version_base"] = "1.1"
+        if version_compare(hydra.__version__, "1.2.0") > 0:
+            hydra_args["version_base"] = "1.2"
```

Comment:
```suggestion
        if version_compare(hydra.__version__, "1.2.0") > 0:
            hydra_args["version_base"] = None
        elif version_compare(hydra.__version__, "1.1.2") > 0:
            hydra_args["version_base"] = "1.1"
```

https://hydra.cc/docs/1.2/upgrades/version_base/ mentions that from hydra 1.2 on setting `version_base` to None chooses automatically the correct defaults.